src/main/java/com/commercetools/project/sync/Syncer.java

```diff
@@ -114,6 +114,8 @@ public Syncer(
     }
 
     return customObjectService
+        .attachRunnerName(runnerName)
```

Comment:
There is a problem with this design -> Now the service is stateful rather stateless. Now each service is bound to one `runnerName` and one `syncModuleName`. But what if we want to get the `getCurrentCtpTimestamp()` of  another runnerName of another sync module? then we always have to attach first?

this is as opposed to `getCurrentCtpTimestamp(runnerName, syncModuleName)` where the service is stateless and only depends on the input given not some predefined value. Do you know what I mean?

Reply:
f71d1a52f166cb28c80d977bc03438630c538a9c